install_config/redeploying_certificates.adoc

```diff
@@ -688,6 +682,18 @@ $ oc create secret tls router-certs --cert=router.pem \ <1>
 <1> *_router.pem_* is the file that contains the concatenation of the
 certificates that you generated.    
 
+. Redeploy the router:
++
+----
+$ oc rollout latest dc/router
+----
+
+When routers are initially deployed, an annotation is added to the router's
+service that automatically creates a
+xref:../dev_guide/secrets.adoc#service-serving-certificate-secrets[service serving certificate secret] named `router-metrics-tls`.
+
```

Comment:
Same as above, this line needs a `+` added so that the ordered list isn't broken.

Reply:
Thank you, I've fixed.